global.json

```
@@ -3,6 +3,7 @@
     "dotnet": "8.0.100-preview.7.23376.3"
   },
   "msbuild-sdks": {
+    "Microsoft.Build.Traversal": "2.0.34",
```

to match what we use in dotnet/runtime:

```suggestion
    "Microsoft.Build.Traversal": "3.4.0",
```